src/app/utils/common.util.ts

```diff
@@ -30,34 +30,55 @@ export class CommonUtil {
     return uniqid;
   }
 
-  static formatBytes(value: number | string): string {
-    const units: readonly string[] = ['kB', 'MB', 'GB', 'TB', 'PB'];
+  static formatMemoryBytes(value: number | string): string {
+    const units: readonly string[] = ['KiB', 'MiB', 'GiB', 'TiB', 'PiB', 'EiB'];
     let unit: string = 'bytes';
```

Comment:
This should be a `B` for consistency with other units.

Reply:
Changed 'bytes' to 'B'.